### DISCUSSION TITLE
Show version of Data API Builder running in Static Web App

### DISCUSSION THREAD

**Discussion by wrutkowski-xebia** [Azure/data-api-builder#2133] [Ideas]
Hello,

Can't find it so maybe this is not implemented.

It would be nice to know on Static Web App hosting, what version of Data API Builder is currently running or something.

Well maybe some other stats as connection with DB and others.

Regards

**Comment by seantleonard** [comment]
Thank you for bringing this up. This feature is in-review via #2086 , and will be in future version of dab.
